milli/src/vector/openai.rs

```diff
@@ -34,13 +34,20 @@ pub struct EmbedderOptions {
 #[serde(deny_unknown_fields, rename_all = "camelCase")]
 #[deserr(rename_all = camelCase, deny_unknown_fields)]
 pub enum EmbeddingModel {
+    // # WARNING
+    //
+    // If ever adding a model, make sure to add it to the list of supported models below.
     #[default]
     #[serde(rename = "text-embedding-ada-002")]
     #[deserr(rename = "text-embedding-ada-002")]
     TextEmbeddingAda002,
 }
 
 impl EmbeddingModel {
+    pub fn supported_models() -> &'static [&'static str] {
+        &["text-embedding-ada-002"]
+    }
```

Comment:
I was thinking about using [this kind of library](https://lib.rs/crates/enum-iterator) to reduce the possibility of bugs, maybe? 🤔 But we need to implement `Display` on the type and we will not be able to return `static str`s.

Reply:
Considering it is rather unlikely to add more models at this point, I think it's best to wait until we add another model to consider enum-iterator. It's a good suggestion, though!